README.md

```diff
@@ -1,2 +1,99 @@
 # bootcamp-ecommerce
 MIT Bootcamp e-commerce
+
+## Major Dependencies
+- Docker
+  - [Download from Docker website](https://docker.com/). Follow the OS X or Linux instructions depending on your OS.
+- [Virtualbox](https://www.virtualbox.org/wiki/Downloads) to run edX locally
+
+
+## Running edX devstack locally
+
+#### 1) Create your edx docker container with at least 4G memory:
+
+    docker-machine create --virtualbox-memory "4096" edx
+    eval $(docker-machine env edx)
```

Comment:
Do you want to recommend that folks make note of the IP address?

Reply:
Sure